forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+  is no great effect but...
+
+
+\begin{resp}
+  The reviewer is correct: the choice of $\tau$ rather than a monotone function of
+  $\tau$ will make a difference. But it seems that we should have been more clear
+  in our description of WIS. The weights used here are those used to evaluate
+  forecasts submitted to the COVID-19 Forecast Hub and the CDC. More broadly,
+  this is a standard metric in the forecasting community (just as Mean Squared
+  Forecast Error is standard), rather than an ``author decision''.  In principle,
+  we would expect that different weights may alter the conclusions, though
+  likely not substantively. Applying a monotone function to tau only effectively
+  changes the quantile of interest, resulting in a mismatch between coverage you
+  tried to get and the coverage WIS is evaluating. Applying the function to $\tau$
+  and $(1-\tau)$ results in an asymmetry, penalizing forecasts that miss on one
+  side more than on the other.
+
+  $\bullet$ We have added a sentence in the Evaluation Metrics section emphasizing that
+  this metric is standard.
+\end{resp}
+
+
+  2. They do not discuss the possible effects of vaccination rates and other
+  factors, though this may be one of the areas of localization they propose to
+  investigate.
+
+
+
+\begin{resp}
+  During the period discussed in the main paper, vaccinations hadn’t really
+  taken off. On Dec 31 2000, when the main evaluation period ends,  according to
+  CDC data, about 1\% of the population 
+  had received 1 dose. That said, we should probably be more specific in the
+  paragraph on lines 31-40: our goal is to use very simple models that do not
+  account for extra information (vaccinations but also strains of the virus,
+  ``super-spreader'' events, etc.).
+
+  $\bullet$ We have added a sentence around lines 30--40 emphasizing the choice
+  to ignore such factors.
+
+  $\bullet$ We also added some text to the Prediction Period section stating
+  that our end 
+  date (31 December) coincides with the introduction of vaccinations.
+\end{resp}
+
+
+  3. This points to what is perhaps a more major issue. Can they identify in
+  quantitative terms the consequences of the additional improvements in
+  forecasting provided by these methods in terms of gains in resources available
+  to deal with the epidemic? Unfortunately as they state, in the crucial periods
+  of sharply rising cases, the additional benefit seems least.
+
+
+\begin{resp}
+  This is a good point, and we have made a few modifications described in more
+  detail below (Reviewer 2, Point 3).
+\end{resp}
+
+
+
+  4. Despite the fact that a detailed description of terms in the text is
+  impossible, it would I think be helpful to say if there is any significance to
+  the names they associate with their first 2 and 4th indicators.
+
+
+\begin{resp}
+  It seems that the Reviewer is referring to CHNG-COVID and DV-CLI. These are
```

Comment:
(Just to be clear) I think the Reviewer is referring to CHNG-COVID, CHNG-CLI, and DV-CLI (it is phrased very weirdly, but they say "first two" and "fourth" indicators